src/schema-extension-api.ts

```diff
@@ -71,20 +71,22 @@ class SchemaExtensionAPI implements ExtensionAPI {
     }
 
 	/**
-	 * Call requestSchema for each provider and find the first one who reports he can provide the schema.
+	 * Call requestSchema for each provider and finds all matches.
 	 *
 	 * @param {string} resource
 	 * @returns {string} the schema uri
 	 */
-	public requestCustomSchema(resource: string): string {
+	public requestCustomSchema(resource: string): string[] {
```

Comment:
Similar to server should we keep compatibility by returning string or string[]?

Reply:
This part is just internal API, it needs to be string[] now so that it can find all matching schema contributes for the given file